src/webui/webui.cpp

```diff
@@ -54,6 +54,7 @@ void WebUI::init()
 
     const quint16 oldPort = m_port;
     m_port = pref->getWebUiPort();
+    const auto address = pref->getWebUiAddress();
```

Comment:
I don't think it's obvious here that `getWebUiAddress()` returns a `QHostAddress`